scaleway/resource_baremetal_server_beta.go

```diff
@@ -223,7 +223,7 @@ func resourceScalewayBaremetalServerBetaUpdate(d *schema.ResourceData, m interfa
 
 	// FIXME: Implement ssh_key_ids changes
 
-	if d.HasChange("os_id") {
+	if d.HasChange("os_id") || d.HasChange("ssh_key_ids") {
 		installReq := &baremetal.InstallServerRequest{
 			Zone:     zone,
 			ServerID: ID,
```

Comment:
`if len(installReq.SSHKeyIDs) ==0` before doing the request, use code at [line 125](https://github.com/terraform-providers/terraform-provider-scaleway/pull/250/files#diff-cb439764a0da1faaa451c0306849d0f9L125)